effects/antidot_effect.py

```diff
@@ -0,0 +1,38 @@
+import message_text_config as msg
+import user_interaction
+import utils
+from citizens.citizen import Citizen
+from citizens.player import Player
+from game import Game
+
+from effects.effect import Effect
+
+
+class AntidotEffect(Effect):
+    def __init__(self, game: Game, name: str, creator: Citizen) -> None:
+        super().__init__(game, name, creator, 12)
+
+    def _activate_impl(self) -> bool:
+        target_number = utils.read_target_number(
+            self.context, msg.AntidoteMessages.ACTIVATION_CHOOSE_TARGET,
+            self._validate)
+        self.targets.append(self.game.citizens[target_number - 1])
+
+        self.user_interaction.save_active(
+            msg.AntidoteMessages.ACTIVATION_SUCCESS.format(
+                self.targets[0].name))
+
+        return True
+
+    def _resolve_impl(self) -> bool:
+        if self.activation_round == self.game.round_number:
+            if self.targets[0].hp - 1 == 0:
```

Comment:
There are lot's of situations when this code won't work.

1) self.activation_round == self.game.round_number:
We need this logic on NEXT round, not current
2) This code won't work if player or citizen receives 2 strikes in one night (it is a valid situation)

My purpose:
expire AntidotEffect next round after it was activated:
smthing like
if self.activation_round == self.game.round_number:
    return False
return True

And in on_clear method check if target is alive
If it is not - set it's helth to 1

Reply:
Ok. Forgot this solution. It won't work without some trics

Reply:
What if we simply change it priority to the lowest possible?
It will guarantee that this effect will be executed last (when we'll already now if target survived the night or not)?

And simply use logic:
if not is_alive():
  hp = 1

Reply:
Done. Now antidot has "-1" priority